indico/modules/rb/client/js/modules/roomList/RoomList.jsx

```diff
@@ -176,6 +204,23 @@ class RoomList extends React.Component {
                     <div styleName="actions">
                       {selectionMode ? (
                         <>
+                          {selectionMode === 'export' ? (
```

Comment:
You could use `&&` instead of using a ternary operator to conditionally render the button (this gets rid of the `undefined`).

```suggestion
                          {selectionMode === 'export' && (
```